package.json

```diff
@@ -4,7 +4,7 @@
   "author": "Tobias Koppers @sokra",
   "description": "Serves a webpack app. Updates the browser on changes.",
   "peerDependencies": {
-    "webpack": ">=2.0.3-beta <3"
+    "webpack": "^2 || 2.0.4-beta || 2.0.5-beta || 2.0.6-beta || 2.0.7-beta || 2.1.0-beta.0"
   },
```

Comment:
Same thing here as my comment on your pull request for fixing babel-loader's package.json

(changing `2.1.0-beta.0` to `^2.1.0-beta.0` would make it also match 2.1.0-beta.1 and 2.1.0-beta.2, which also has been released now)